app/src/main/res/layout/completed_story_item.xml

```diff
@@ -1,81 +1,173 @@
-        android:contentDescription="@{viewModel.completedStory.storyName}"
-        android:scaleType="centerInside"
-        android:src="@{viewModel.completedStory.lessonThumbnail.thumbnailGraphic}"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintDimensionRatio="16:9"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:ellipsize="end"
+        android:fontFamily="sans-serif"
+        android:gravity="top"
+        android:maxLines="2"
+        android:minLines="2"
+        android:text="@{viewModel.completedStory.storyName}"
+        android:textColor="@color/oppiaPrimaryText"
+        android:textSize="14sp"
+        android:textStyle="bold"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:roundedRectDrawableWithColor="@{viewModel.completedStory.lessonThumbnail.backgroundColorRgb}" />
+        app:layout_constraintTop_toTopOf="parent" />
 
-      <LinearLayout
-        android:layout_width="0dp"
+      <TextView
+        android:id="@+id/completed_story_topic_name_text_view"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="12dp"
-        android:layout_marginEnd="12dp"
-        android:orientation="vertical"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toEndOf="@id/completed_story_lesson_thumbnail"
-        app:layout_constraintTop_toTopOf="parent">
-
-        <TextView
-          android:id="@+id/completed_story_name_text_view"
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:layout_marginBottom="4dp"
-          android:ellipsize="end"
-          android:fontFamily="sans-serif"
-          android:maxLines="2"
-          android:text="@{viewModel.completedStory.storyName}"
-          android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp" />
-
-        <TextView
-          android:id="@+id/completed_story_topic_name_text_view"
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:ellipsize="end"
-          android:fontFamily="sans-serif-light"
-          android:maxLines="1"
-          android:text="@{viewModel.completedStory.topicName}"
-          android:textAllCaps="true"
-          android:textColor="@color/oppiaPrimaryText"
-          android:textSize="14sp" />
-      </LinearLayout>
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="4dp"
+        android:layout_marginEnd="8dp"
+        android:fontFamily="sans-serif-light"
+        android:paddingBottom="12dp"
+        android:textAllCaps="true"
+        android:text="@{viewModel.completedStory.topicName}"
+        android:textColor="@color/oppiaStrokeBlack"
```

Comment:
Introduce max lines and min lines for this text view.

Reply:
Added max and min lines.